packages/data-context/src/DataContext.ts

```diff
@@ -132,10 +132,6 @@ export class DataContext {
     return this._config.localSettingsApi
   }
 
-  get isGlobalMode () {
-    return this.appData.isInGlobalMode
-  }
```

Comment:
In the existing code, the variable `isGlobalMode` was used to expose `isInGlobalMode`. This led to some confusion and was an unnecessary duplication, so I removed it in favor of using `this.ctx.modeOptions.global` in the one location where `this.ctx.isGlobalMode` was being used